src/test/HostActivationTests/DotNetBuilder.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.DotNet.Cli.Build;
+using System;
+using System.IO;
+
+namespace Microsoft.DotNet.CoreSetup.Test.HostActivation
+{
+    public class DotNetBuilder
```

Comment:
Can you please add a comment that this uses mock `hostpolicy`? So it can't be used to actually run any apps.
Or maybe a general description of all the mocks this uses...

Eventually we should come up with a better name (or rename the methods to make it more explicit which ones are mocking) - right now I don't have a good idea though.